templates/react/components/foo/Show.js

```diff
@@ -62,7 +62,7 @@ class Show extends Component {
       </div>
       }
       <Link to=".." className="btn btn-default">Back to list</Link>
-      {item && <Link to={`/{{ lc }}/edit/${encodeURIComponent(item['@id'])}`}>
+      {item && <Link to={`/{{ name }}/edit/${encodeURIComponent(item['@id'])}`}>
```

Comment:
this is change becasue routings changed in routes/foo.js from {lc} to {name} which is later used in pagination